src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    @include square($star-size);
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)